test/ilib-lint-plugin-test/src/TestParser.js

```diff
@@ -39,6 +39,9 @@ class TestParser extends Parser {
     }
 
     parseData(data) {
+        if (!data) {
+            throw "ilib-lint-plugin-test: attempt to parse empty data";
```

Comment:
Minor nitpick: one should not throw strings directly, use `new Error`

Reply:
fixed in the subsequent commit